Library/Homebrew/brew.rb

```diff
@@ -141,9 +141,14 @@ class MissingEnvironmentVariables < RuntimeError; end
       end
       tap_commands += %W[#{HOMEBREW_BREW_FILE} tap #{possible_tap.name}]
       safe_system(*tap_commands)
+      ohai "Automatically tapped #{possible_tap.name}. To remove, run `brew untap #{possible_tap.name}`"
```

Comment:
The output indicates it was tapped so don't think this is necessary.

Reply:
Fair, but it does sound like there is some confusion based on the linked issue. However, it's only one person (that I've seen) so I'm willing to remove it. If it comes up more in the future I'd ask that we take another look.